### ISSUE TITLE
Ability to group flights to toggle between and isolate a flight path.

### ISSUE THREAD

**Issue by in1ka** [smrghsh/drones#12]
We need the ability to group flights
 - in a similar location. 
 - menu should appear to toggle between them. 

For implementation, we will need to: 
- standardize a `Flight.js` class so every unit of contiguous drone data has a standardized API for visualization and analysis. Flights should have the pathway, telemetry data, placard of metadata, and associated photogrammetry/sensor data as well.
- create a `FlightGroup.js` class that can contain Flights and position a menu nearby in 3D where clicking on buttons can toggle visibility.
